lib/tasks/attachments.rake

```diff
@@ -0,0 +1,14 @@
+namespace :attachments do
```

Comment:
Allows us to catch processing attemps that have failed as sucker punch won’t retry an item. We can set this to run hourly